modules/installation-guide/partials/proc_installing-che-on-kubernetes_using_chectl_and_helm.adoc

```diff
@@ -0,0 +1,100 @@
+  ✔ 👀  Looking for an already existing {prod} instance
+    ✔ Verify if {prod} is deployed into namespace "{prod-namespace}"...it is not
+  ✔ ✈️  {kubernetes} preflight checklist
+    ✔ Verify if kubectl is installed
+    ✔ Verify remote kubernetes status...done.
+    ✔ Check {kubernetes} version: Found v1.15.12-gke.2.
+    ✔ Verify domain is set...set to {domain}.
+    ↓ Check if cluster accessible [skipped]
+  ✔ Following {prod} logs
+    ↓ Start following Operator logs [skipped]
+    ✔ Start following {prod} server logs...done
+    ✔ Start following Postgres logs...done
+    ✔ Start following Keycloak logs...done
+    ✔ Start following Plugin registry logs...done
+    ✔ Start following Devfile registry logs...done
+    ✔ Start following namespace events...done
+  ✔ 🏃‍  Running Helm to install {prod}
+    ✔ Verify if helm is installed
+    ✔ Check Helm Version: Found v3.4.1+gc4e7485
+    ✔ Create Namespace ({prod-namespace})...does already exist.
+    ✔ Check {prod} TLS certificate...self-signed TLS certificate secret found
+    ✔ Check Cluster Role Binding...does not exists.
+    ✔ Preparing {prod} Helm Chart...done.
+    ✔ Updating Helm Chart dependencies...done.
+    ✔ Deploying {prod} Helm Chart...done.
+  ✔ ✅  Post installation checklist
+    ✔ PostgreSQL pod bootstrap
+      ✔ Scheduling...done
+      ✔ Downloading images...done
+      ✔ Starting...done
+    ✔ Devfile registry pod bootstrap
+      ✔ Scheduling...done
+      ✔ Downloading images...done
+      ✔ Starting...done
+    ✔ Plugin registry pod bootstrap
+      ✔ Scheduling...done
+      ✔ Downloading images...done
+      ✔ Starting...done
+    ✔ {prod} pod bootstrap
+      ✔ Scheduling...done
+      ✔ Downloading images...done
+      ✔ Starting...done
+    ✔ {prod} status check...done
+  ✔ Prepare post installation output...done
+  ✔ Show important messages
+    ✔ {prod} {prod-ver} has been successfully deployed.
+    ✔ Documentation             : https://www.eclipse.org/che/docs/
+    ✔ -------------------------------------------------------------------------------
+    ✔ Users Dashboard           : https://{prod-namespace}-{prod-namespace}.{domain}
+    ✔ Admin user login          : "XXX:XXX". NOTE: must change after first login.
+    ✔ -------------------------------------------------------------------------------
+    ✔ Plug-in Registry          : https://plugin-registry-{prod-namespace}.{domain}/v3
+    ✔ Devfile Registry          : https://devfile-registry-{prod-namespace}.{domain}
+    ✔ -------------------------------------------------------------------------------
+    ✔ Identity Provider URL     : https://keycloak-{prod-namespace}.{domain}/auth
+    ✔ Identity Provider login   : "XXX:XXX".
+    ✔ -------------------------------------------------------------------------------
+----
+
+.Verification steps
+
+. Track installation events with:
++
+[subs="+attributes"]
+----
+$ kubectl get events --namespace {prod-namespace} --sort-by=.metadata.creationTimestamp
+----
```

Comment:
```suggestion
. Investigate {prod} logs:
+
[subs="+attributes"]
----
$ {prod-cli} server:logs
----
```

Reply:
Does it display Che Server logs only?
Will it display PVC attaching issues and other?

Reply:
It collects logs + namespace events

Reply:
Let me apply suggested changes then

Reply:
Done.

For other reviewers to see difference:

chectl server:logs actually shows nothing but prepare a place where user can take a look for details
```
~/projects/che-docs reuseChectlK8sCommands > chectl server:logs -n dashboard-next                                                                              INT 17:08:02
› Current Kubernetes context: 'test/api-cluster-devtools-954b-devtools-954b-example-opentlc-com:6443/opentlc-mgr'
Eclipse Che logs will be available in '/tmp/chectl-logs/1608044891171'
  ✔ Verify Kubernetes API...OK (it's OpenShift)
  ✔ Verify if namespace 'dashboard-next' exists
  ✔ Read Operator logs...done
  ✔ Read Eclipse Che server logs...done
  ✔ Read Postgres logs...done
  ✔ Read Keycloak logs...done
  ✔ Read Plugin registry logs...done
  ✔ Read Devfile registry logs...done
  ✔ Read namespace events...done
Command server:logs has completed successfully in 00:04.
...

/tmp/chectl-logs/1608044891171/dashboard-next > tree .                                                                                                          5s 17:22:25
.
├── che-7bcb5d9f99-v4l86
│   └── che.log
├── che-operator-f8f98b559-gxjw4
│   └── che-operator.log
├── devfile-registry-775f9c6cdb-h9h4b
│   └── che-devfile-registry.log
├── events.txt
├── keycloak-745f85b998-d4v5z
│   └── keycloak.log
├── plugin-registry-99654747d-x7jvf
│   └── che-plugin-registry.log
└── postgres-67b9d78d6-6qn4t
    └── postgres.log

6 directories, 7 files
```


kubectl get events:
```
~/projects/che-docs reuseChectlK8sCommands > kubectl get events --namespace che --sort-by=.metadata.creationTimestamp                            kube minikube/che 17:22:55
LAST SEEN   TYPE      REASON                  OBJECT                                     MESSAGE
2m25s       Normal    Issuing                 certificate/che-certificate                Issuing certificate as Secret does not exist
2m25s       Normal    Issuing                 certificate/che-certificate                The certificate has been successfully issued
2m25s       Normal    Requested               certificate/che-certificate                Created new CertificateRequest resource "che-certificate-hj589"
2m25s       Normal    CertificateIssued       certificaterequest/che-certificate-hj589   Certificate fetched from issuer successfully
2m25s       Normal    Generated               certificate/che-certificate                Stored new private key in temporary Secret resource "che-certificate-htc6r"
2m4s        Normal    ExternalProvisioning    persistentvolumeclaim/che-data-volume      waiting for a volume to be created, either by external provisioner "k8s.io/minikube-hostpath" or manually created by system administrator
2m4s        Normal    CREATE                  ingress/devfile-registry                   Ingress che/devfile-registry
2m4s        Normal    ScalingReplicaSet       deployment/che                             Scaled up replica set che-59b9466fbf to 1
2m4s        Normal    Provisioning            persistentvolumeclaim/che-data-volume      External provisioner is provisioning volume for claim "che/che-data-volume"
2m4s        Normal    ScalingReplicaSet       deployment/plugin-registry                 Scaled up replica set plugin-registry-6564d96c4f to 1
2m4s        Normal    SuccessfulCreate        replicaset/plugin-registry-6564d96c4f      Created pod: plugin-registry-6564d96c4f-xzk2g
2m4s        Normal    ProvisioningSucceeded   persistentvolumeclaim/che-data-volume      Successfully provisioned volume pvc-2b1c01e8-58d9-4556-8940-91c885429318
2m3s        Normal    ScalingReplicaSet       deployment/devfile-registry                Scaled up replica set devfile-registry-598ffcf745 to 1
2m3s        Normal    SuccessfulCreate        replicaset/che-59b9466fbf                  Created pod: che-59b9466fbf-v5q5s
2m3s        Normal    Scheduled               pod/plugin-registry-6564d96c4f-xzk2g       Successfully assigned che/plugin-registry-6564d96c4f-xzk2g to minikube
2m3s        Normal    Scheduled               pod/che-dashboard-65f7b5b9-zbtzn           Successfully assigned che/che-dashboard-65f7b5b9-zbtzn to minikube
2m3s        Normal    SuccessfulCreate        replicaset/devfile-registry-598ffcf745     Created pod: devfile-registry-598ffcf745-q9q4l
2m3s        Normal    Scheduled               pod/devfile-registry-598ffcf745-q9q4l      Successfully assigned che/devfile-registry-598ffcf745-q9q4l to minikube
2m3s        Normal    ScalingReplicaSet       deployment/che-dashboard                   Scaled up replica set che-dashboard-65f7b5b9 to 1
2m4s        Normal    CREATE                  ingress/plugin-registry                    Ingress che/plugin-registry
2m3s        Normal    SuccessfulCreate        replicaset/che-dashboard-65f7b5b9          Created pod: che-dashboard-65f7b5b9-zbtzn
2m4s        Normal    CREATE                  ingress/che-dashboard-ingress              Ingress che/che-dashboard-ingress
2m4s        Normal    CREATE                  ingress/che-ingress                        Ingress che/che-ingress
2m3s        Normal    Scheduled               pod/che-59b9466fbf-v5q5s                   Successfully assigned che/che-59b9466fbf-v5q5s to minikube
2m2s        Warning   FailedMount             pod/plugin-registry-6564d96c4f-xzk2g       MountVolume.SetUp failed for volume "default-token-r9bh7" : failed to sync secret cache: timed out waiting for the condition
2m2s        Normal    Pulling                 pod/che-59b9466fbf-v5q5s                   Pulling image "busybox"
2m2s        Warning   FailedMount             pod/devfile-registry-598ffcf745-q9q4l      MountVolume.SetUp failed for volume "default-token-r9bh7" : failed to sync secret cache: timed out waiting for the condition
2m1s        Normal    Pulling                 pod/plugin-registry-6564d96c4f-xzk2g       Pulling image "quay.io/eclipse/che-plugin-registry:nightly"
2m          Normal    Pulled                  pod/che-59b9466fbf-v5q5s                   Successfully pulled image "busybox" in 2.157914041s
2m          Normal    Started                 pod/che-59b9466fbf-v5q5s                   Started container fmp-volume-permission
2m          Normal    Created                 pod/che-59b9466fbf-v5q5s                   Created container fmp-volume-permission
2m          Normal    Pulling                 pod/che-dashboard-65f7b5b9-zbtzn           Pulling image "quay.io/eclipse/che-dashboard:next-react"
2m          Normal    Pulling                 pod/devfile-registry-598ffcf745-q9q4l      Pulling image "quay.io/eclipse/che-devfile-registry:nightly"
119s        Normal    Pulling                 pod/che-59b9466fbf-v5q5s                   Pulling image "quay.io/eclipse/che-server:nightly"
116s        Normal    Pulled                  pod/plugin-registry-6564d96c4f-xzk2g       Successfully pulled image "quay.io/eclipse/che-plugin-registry:nightly" in 4.742623408s
115s        Normal    Started                 pod/plugin-registry-6564d96c4f-xzk2g       Started container che-plugin-registry
115s        Normal    Created                 pod/plugin-registry-6564d96c4f-xzk2g       Created container che-plugin-registry
114s        Normal    Pulled                  pod/che-dashboard-65f7b5b9-zbtzn           Successfully pulled image "quay.io/eclipse/che-dashboard:next-react" in 6.83087131s
114s        Normal    Created                 pod/che-dashboard-65f7b5b9-zbtzn           Created container che-dashboard
113s        Normal    Started                 pod/che-dashboard-65f7b5b9-zbtzn           Started container che-dashboard
112s        Normal    Created                 pod/devfile-registry-598ffcf745-q9q4l      Created container che-devfile-registry
112s        Normal    Pulled                  pod/devfile-registry-598ffcf745-q9q4l      Successfully pulled image "quay.io/eclipse/che-devfile-registry:nightly" in 8.286580432s
112s        Normal    Started                 pod/devfile-registry-598ffcf745-q9q4l      Started container che-devfile-registry
110s        Normal    Pulled                  pod/che-59b9466fbf-v5q5s                   Successfully pulled image "quay.io/eclipse/che-server:nightly" in 8.884081s
110s        Normal    Created                 pod/che-59b9466fbf-v5q5s                   Created container che
110s        Normal    Started                 pod/che-59b9466fbf-v5q5s                   Started container che
65s         Normal    UPDATE                  ingress/che-ingress                        Ingress che/che-ingress
64s         Normal    UPDATE                  ingress/devfile-registry                   Ingress che/devfile-registry
64s         Normal    UPDATE                  ingress/plugin-registry                    Ingress che/plugin-registry
64s         Normal    UPDATE                  ingress/che-dashboard-ingress              Ingress che/che-dashboard-ingress
```

Personally I see a bit instance and handy to use, but it's the issue server:logs can address